granulate_utils/linux/cgroups/memory_cgroup.py

```diff
@@ -22,23 +24,23 @@ def get_max_usage_in_bytes(self) -> int:
     def get_usage_in_bytes(self) -> int:
         return int(self.read_from_control_file(self.usage_in_bytes))
 
-    def _set_memsw_limit_in_bytes(self, limit: int) -> None:
+    def _set_memsw_limit_in_bytes(self, controller_path: Path, limit: int) -> None:
```

Comment:
Previously the class worked on my own process. Now we wanna let it work on a path but please keep the class encapsulation i.e have `self.cgroup` point to the relevant cgroup that will be set in the c'tor.

So the usage would be:
```
cg = CpuCgroup("/docker/aaaaaaa")
cg.set_cpu_limit_cores(0.5)
```

for all methods

Reply:
Closing this PR and opened another https://github.com/Granulate/granulate-utils/pull/117 .